main.py

```diff
@@ -1 +1,23 @@
 #Mad Libs Generator Project
+//Loop back to this point once code finishes
+loop = 1
+while (loop < 9):
```

Comment:
Let's take another look at this while loop. The requirements say to loop through 10 times. While loops don't use parenthesis.
```suggestion
while loop <= 10:
```